### PR TITLE
Cngfw azure terraform

### PR DESCRIPTION
**Pull request by abudilovskiy-panw**
## Description

Updating the Cloud NGFW sections with Cloud NGFW for Azure information and a basic tutorial
## Types of changes

- New feature (non-breaking change which adds functionality)
- 
## Checklist

- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (migara) [inline products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:46]:
```suggestion
panorama-string = "Panorama registration string"
```

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (migara) [inline products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:76]:
```suggestion
1. Resource Group
```

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (migara) [inline products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:37]: Do we need a subtitle here? ##Setup ?

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (migara) [inline products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:38]:
```suggestion
First, in a separate directory create base Terraform files: `main.tf``, `variables.tf`` and `terraform.tfvars`.
```

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (migara) [inline products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:94]: In the objectives section we mentioned we are going to create 2 x PIPs. Perhaps remove "at least one IP address"?

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (migara) [inline products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:180]: Should we add "Panorama instance with the Azure plugin installed" as a pre-req in the begining of this article. 

Also if the registration string == panorama string probably a good idea use one attribute name throughout to avoid confusion

File: products/cloudngfw/docs/getstarted_azure.mdx
Comment (migara) [inline products/cloudngfw/docs/getstarted_azure.mdx:38]:
```suggestion
For a description of the Cloud NGFW for Azure APIs, see the [API Reference](/cloudngfw/azure/api).
```

File: products/cloudngfw/docs/getstarted_azure.mdx
Comment (migara) [inline products/cloudngfw/docs/getstarted_azure.mdx:35]:
```suggestion
- **Azure Portal** is a graphical user interface that provides a native experience similar to any other Azure service.
```

File: products/cloudngfw/docs/getstarted_azure.mdx
Comment (migara) [inline products/cloudngfw/docs/getstarted_azure.mdx:24]:
```suggestion
In the first (Azure Rulestack managed) option, you create rules based on matching criteria and leverage the Security Services straight from Azure Portal. 
```

File: products/cloudngfw/docs/getstarted_azure.mdx
Comment (migara) [inline products/cloudngfw/docs/getstarted_azure.mdx:18]:
```suggestion
- **Subscribe to the Cloud NGFW Service**—Begin by subscribing to the Cloud NGFW for Azure service through the Azure Marketplace. You can also search for "Cloud NGFW" in the Azure Services to locate Local Rulestack and Cloud NGFW resources. The service relies on your Entre ID and IAM permissions and generally requires a "Contributor" role on the subscription.
```

**Review by migara — APPROVED** [review]
Great work! 🚀

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (migara) [inline products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:101]:
```suggestion
  sku                 = "Standard"
```

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (migara) [inline products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:109]:
```suggestion
  sku                 = "Standard"
```

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (abudilovskiy-panw) [reply products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:46]: Added

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (abudilovskiy-panw) [reply products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:189]: Spelled out the attr

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (abudilovskiy-panw) [reply products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx]: fixed

**Comment by abudilovskiy-panw** [comment]
Thanks for the thorough review @migara  I've resolved the comments.

File: products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx
Comment (migara) [inline products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:199]: One quick thing I noticed - `data.azurerm_virtual_hub.vhub-westeu.id` should this be `azurerm_virtual_hub.vhub-westeu.id` instead?

Comment (migara) [reply products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:199]: Same on line 214 and 232

Comment (abudilovskiy-panw) [reply products/terraform/docs/cloudngfw/azure/tutorials/intro.mdx:199]: Nice catch! I was testing with the existing hub and that's the leftovers. Fixed.

**Comment by abudilovskiy-panw** [comment]
@migara anything else needed to merge this?

**Comment by migara** [comment]
> @migara anything else needed to merge this?

@abudilovskiy-panw LGTM, feel free to merge it when you're ready!

**Comment by abudilovskiy-panw** [comment]
@migara looks like I don't have write access to do the merge

**Comment by jamesholland-uk** [comment]
Done 👍

**Pull request merged**
